core/src/main/java/org/sonarsource/sonarlint/core/container/analysis/filesystem/SonarLintInputFile.java

```diff
@@ -245,4 +249,28 @@ public boolean hasNoSonarAt(int line) {
     return this.noSonarLines.contains(line);
   }
 
+  public boolean isIgnoreAllIssues() {
+    checkMetadata();
```

Comment:
@dbmeneses this `checkMetadata()` statement is missing in the scanner-engine, and I think it is a bug. Thanks to the medium test, I found that if the first issue reported by the analyzer is a file level issue (like squid:S1220), then metadata are not initialized, and so the `ignoreAllFiles` boolean has not been set.

Reply:
Don't forget to create the same ticket in SC